easybuild/easyconfigs/g/GBprocesS/GBprocesS-2.3-intel-2020a-Python-3.8.2.eb

```diff
@@ -38,6 +39,20 @@ pretestopts = (
 
 runtest = 'python -m unittest discover -v -s ./test -p test_*.py'
 
+sanity_check_commands = [
+    'export TEMP=$(mktemp -d)'
+    ''' && echo > $TEMP/config.ini <<EOS
+
+# Main block General
+cores = 1
+input_directory = /data/run/
+sequencing_type = pe
+input_file_name_template = {run:24}_R{orientation:1}_001{extension:10}
+temp_dir = /tmp/
+EOS'''
```

Comment:
I think this deserves being formatted slightly different:

```python
# configuration file to use when running gbprocess sanity check
# see <ADD POINTER TO GBPROCESS DOCS HERE>
local_gbprocess_cfg = """
# Main block General
cores = 1
input_directory = /data/run/
sequencing_type = pe
input_file_name_template = {run:24}_R{orientation:1}_001{extension:10}
temp_dir = %(builddir)s/
"""

sanity_check_commands = [
    "echo > %(builddir)s/config.ini <<EOF\n" + local_gbprocess_cfg + "\nEOF",
    "gbprocess --debug -c %(builddir)s/config.ini",
]

Reply:
Yup, makes sense.